### PR TITLE
OSASINFRA-3574: Update worker nodes in OVN PerfScale ingress-perf job

### PR DESCRIPTION
**Pull request by MiguelCarpio**
Updating worker and infra nodes:

OpenStack Nodes Config (proposal)

Node Types:
- Master: m1.xlarge 
- Worker: m1.xlarge 
- Workload: ingress-perf
- Infra: m1.xlarge 

Node Counts:
- Master: 3
- Worker: 4
- Infra: 3

### CODE REVIEW

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

File: ci-operator/config/shiftstack/ci/shiftstack-ci-release-4.16.yaml [hunk 1]
```diff
@@ -203,6 +203,9 @@ tests:
   minimum_interval: 336h
   steps:
     cluster_profile: openstack-vh-mecha-az0
+    env:
+      INFRA_NODES_TO_TAG: "3"
```
Comment (mandre) [inline ci-operator/config/shiftstack/ci/shiftstack-ci-release-4.16.yaml:207]: Why do we need to differ from the default of `2` infra nodes to tag?

**Review by mandre — CHANGES_REQUESTED** [review]
Can you squash the commits?

Then I think this is good to go.

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-release-4.16-e2e-openstack-ingress-perf

**Review by mandre — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**